src/components/Combobox/components/ComboboxRenderers/ComboboxRenderers.jsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import cx from "classnames";
+import ComboboxOption from "components/Combobox/components/ComboboxOption/ComboboxOption";
+import ComboboxCategory from "components/Combobox/components/ComboboxCategory/ComboboxCategory";
+import Divider from "../../../Divider/Divider";
+import { COMBOBOX_DIVIDER_ITEM, COMBOBOX_CATEGORY_ITEM, COMBOBOX_OPTION_ITEM } from "../ComboboxConstants";
+import styles from "./ComboboxRenderers.module.scss";
+
+const DIVIDER_HEIGHT = 17;
+const CATEGORY_HEIGHT = 32;
+
+export function createDividerItemObject({ categoryId }) {
+  return { type: COMBOBOX_DIVIDER_ITEM, height: DIVIDER_HEIGHT, id: `${categoryId}-divider` };
+}
+
+export function createCategoryItemObject({ withDivider, categoryId, categoryData }) {
+  return {
+    height: CATEGORY_HEIGHT,
+    type: COMBOBOX_CATEGORY_ITEM,
+    category: categoryData,
+    id: categoryId,
+    withDivider
+  };
+}
+
+export function createOptionItemObject({
+  option,
+  height,
+  index,
+  optionRenderer,
+  isActive,
+  isActiveByKeyboard,
+  onOptionClick,
+  onOptionEnter,
+  onOptionLeave,
+  optionLineHeight,
+  shouldScrollToSelectedItem,
+  categoryId
+}) {
+  return {
+    type: COMBOBOX_OPTION_ITEM,
+    height,
+    belongToCategory: true,
+    index,
+    option,
+    id: option.id || index,
+    optionRenderer,
+    isActive,
+    isActiveByKeyboard,
+    onOptionClick,
+    onOptionEnter,
+    onOptionLeave,
+    optionLineHeight,
+    shouldScrollToSelectedItem,
+    categoryId
+  };
+}
+
+export function comboboxItemRenderer({ item, _index, style, optionEvents, optionRenderData, isVirtualized }) {
+  const { type, ...otherArgs } = item;
+  let customClassNames;
+  let innerElement;
+  switch (type) {
+    case COMBOBOX_DIVIDER_ITEM: {
+      innerElement = dividerItemRenderer(otherArgs);
+      break;
+    }
+    case COMBOBOX_CATEGORY_ITEM: {
+      innerElement = categoryItemRenderer(otherArgs);
+      if (!isVirtualized) {
+        customClassNames = styles.sticky;
```

Comment:
I'm not sure how it works and how standard it to import from scss file

Reply:
that's the standard way to use css modules :)

Reply:
css modules replace the class name (styles.name) with longer unique name with more parameters according to configurations